ext/configuration.h

```diff
@@ -95,7 +95,6 @@ enum ddtrace_sampling_rules_format {
     CONFIG(BOOL, DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED, "true")                                               \
     CONFIG(BOOL, DD_HTTP_SERVER_ROUTE_BASED_NAMING, "true")                                                    \
     CONFIG(SET, DD_INTEGRATIONS_DISABLED, "default")                                                           \
-    CONFIG(BOOL, DD_PRIORITY_SAMPLING, "true")                                                                 \
```

Comment:
Why are we removing this config? Was it only used with PHP5??

Reply:
Yes, it's completely unused.